packages/react-components/theme-designer/src/ThemeDesigner.styles.ts

```diff
@@ -1,4 +1,11 @@
-import { makeStyles } from '@griffel/react';
+import { makeStaticStyles, makeStyles } from '@griffel/react';
```

Comment:
_nit:_ Import from `@fluentui/react-components`

```suggestion
import { makeStaticStyles, makeStyles } from '@fluentui/react-components';
```